### ISSUE TITLE
Unit tests with coverage failed due to KUBE_TEST_API variable unset

### ISSUE THREAD

**Issue by sshukun** [kubernetes/kubernetes#87075]
**What happened**:
```
$ make test KUBE_COVER=y
hack/make-rules/test.sh: line 277: KUBE_TEST_API: unbound variable
!!! [0110 20:03:49] Call tree:
!!! [0110 20:03:49]  1: hack/make-rules/test.sh:368 runTests(...)
make: *** [Makefile:185: test] Error 1

```
**What you expected to happen**:

unit tests with coverage succeed

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by tedyu** [comment]
Looking at :
https://github.com/kubernetes-sigs/service-catalog/blob/master/pkg/apis/servicecatalog/testapi/testapi.go#L83

It is expecting group version string.

**Comment by liggitt** [comment]
fixed in #87082, clean up from #86256
